test/unit/core.js

```diff
@@ -1215,6 +1215,19 @@ test("jQuery.each(Object,Function)", function() {
 		i++;
 	});
 	equal( i, 2, "Iteration over document.styleSheets" );
+
+	// Trigger JIT
+	for ( i = 0; i < 1000; i++ ) {
```

Comment:
Maybe it would be better to put those assertions in separate test? In my experience, the lesser assertions in one test the better isolation when assertion(s) fails.